### PR TITLE
[SDNTB-15] [SDNTB-159] Create Scanpix search provider

### PR DESCRIPTION
**Pull request by jerome-poisson**
Added Scanpix as a search provider.

Scanpix hide some features (starred and modified time) which are not used

New features are added:

- B&W
- clear edge
- search by one or several ID(s)
- subscription

Some improvments in search provider creation:

- changed placeholder, was showing username and password, now show a proper message
- password use a password field instead of a clear text one
- source is hidden for Scanpix as it is implicit and can disturb the user
 
SDNTB-15

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: scripts/superdesk-search-providers/views/search-provider-config.html [hunk 1]
```diff
@@ -54,19 +54,23 @@ <h3 translate ng-hide="provider._id">Add New Search Provider</h3>
                     </select>
                 </div>
 
-                <div class="field">
+                <div class="field" ng-if="provider.search_provider !== 'scanpix'">
                     <label for="provider-source" translate>Source</label>
-                    <input type="text" id="provider-source" placeholder="{{ source | translate }}" ng-model="provider.source" required>
+                    <input type="text" id="provider-source" placeholder="source of the search provider" ng-model="provider.source" required>
```
Comment (petrjasek) [inline scripts/superdesk-search-providers/views/search-provider-config.html]: here the placeholder is not translatable

**Comment by petrjasek** [comment]
:+1:

**Comment by jerome-poisson** [comment]
fixed, it the build pass travis, this can be merged.

**Comment by jerome-poisson** [comment]
Test are OK, can it be merged?

**Pull request merged**